scripts/datenlord.yaml

```diff
@@ -221,6 +221,9 @@ spec:
               value: csi-etcd-svc:2379
             - name: FUSE_VOLUME_INFO
               value: https://mybucket
+              # The copacity for the cache in memfs
```

Comment:
typo capacity. Actually I mean to provide a comment for the long number " 10737418240" like "request 10G cache"